src/Snippet.tsx

```diff
@@ -20,41 +22,20 @@ interface ISnippetProps {
   lang: string;
 }
 
-enum Highlight {
-  None = 1,
-  Good,
-  Bad,
-}
-
-function highlightCssClass(h: Highlight) {
-  return Highlight[h].toLowerCase();
-}
-
-interface ISnippetState {
-  highlight: Highlight[];
-}
-
-class Snippet extends React.Component<ISnippetProps, ISnippetState> {
+class Snippet extends React.Component<ISnippetProps, {}> {
   constructor(props: ISnippetProps) {
     super(props);
-    const hls = [];
-    for (const _ of props.blocks) {
-      hls.push(Highlight.None);
-    }
-    this.state = { highlight: hls }
+    this.state = {};
   }
 
   public render() {
-    const rb = [];
-    const langclass = "language-" + this.props.lang;
-    const classes = langclass + " block";
+    const childs = [];
```

Comment:
Nit: not a fan of "childs" as a variable name. It's a bit unclear.

Reply:
Do you prefer children?